docs/guide/protocols/relativehybridtopology.rst

```diff
@@ -81,6 +81,56 @@ Each Protocol simulation Unit carries out the following steps:
 
 Note: three different types of multistate sampling (i.e. replica swapping between lambda states) methods can be chosen; HREX, SAMS, and independent (no lambda swaps attempted). By default the HREX approach is selected, this can be altered using ``simulation_settings.sampler_method`` (default: ``repex``).
 
+Analysis
+~~~~~~~~
+
+As standard, some analysis of the each simulation repeat is performed.
+This analysis is made available through either the dictionary of results in the execution output,
+or through some ready-made plots for quick inspection.
+This analysis can be categorised as relating
+to the energetics of the different lambda states that were sampled,
+or to the analysis of the change in structural conformation over time in each state.
+
+.. todo: issue 792, consolidate this page into its own analysis page and link both RBFE and AFE pages to it
+
+* Energetic and replica exchange analysis.  These analyses consider the swapping and energetic overlap between the
+  different simulated states to help assess the convergence and correctness of the estimate of free energy
+  difference produced.
+
+  * MBAR overlap matrix.  This plot is used to assess if the different lambda states simulated overlapped energetically.
+    Each matrix element represents the probability of a sample from a given row state being observable in a given column
+    state.
+    This plot should show that the diagonal of the matrix has some "width" so that the two end states are connected,
+    with off-diagonal elements being at least 0.03[2]_.
+  * Replica exchange overlap matrix (for repex type simulations only).  Similar to the MBAR overlap matrix, this shows
+    the probability of a given lambda state being exchanged with another.  Again, the diagonal of this matrix should be
+    at least tridiagonal wide for the two end states to be connected.
+  * Timeseries of replica states.  This plot shows the time evolution of the different system configurations as they are
+    exchanged between different lambda states.
+    This plot should show that the states are freely mixing and that there are no cliques forming.
+  * Forward and reverse convergence of free energy estimates.  Using increasingly larger portions of the total data,
+    this analysis calculates the free energy estimate, and is repeated both forwards and backwards in time.
+    To assess convergence the forward and reverse estimates should agree within error using only a fraction of the total
+    data[2]_.
+
+* Structural analysis. If a protein was present, these analyses first center and align the system so that
+  the protein is considered the frame of reference.
+
+  * Ligand RMSD.  This produces a plot called ``ligand_RMSD.png`` and a results entry ``ligand_RMSD`` which gives the
+    RMSD of the ligand molecule over time relative to the first frame, for each simulated state.  In correct simulations
```

Comment:
Is this the first frame of the production run or the initial input pdb structure (as first frame)?

Reply:
clarified this, it's of the production run.  So if it rearranged significantly in equilibration this wouldn't cause a large RMSD value.